terra_layer/serializers.py

```diff
@@ -39,6 +48,14 @@ def get_icon(self, obj):
     class Meta:
         model = Scene
         fields = "__all__"
+        extra_kwargs = {"baselayer": {"allow_empty": True}}
+
+    def to_internal_value(self, data):
+        baselayer = data.get("baselayer")
+        querydict = data.copy()
```

Comment:
You should copy only if type base layer is str, no ?

Reply:
Yes, you're right. I did it that way to have only one "return variable". I will change it

Reply:
Done.